.env.sample

```diff
@@ -7,10 +7,12 @@
 HARBOR_USER='admin'
 HARBOR_PASSWORD=''
 HARBOR_BASE_URL='http://127.0.0.1:8083/api/v2.0'
+HARBOR_BASE_REPO_URL='harbor.dev.eks.otomi.cloud'
 OIDC_CLIENT_SECRET=''
 OIDC_ENDPOINT='https://keycloak.dev.gke.otomi.cloud'
-OIDC_VERIFY_CERT='true' # set to false when staging certs are used on the cluster
```

Comment:
Why remove the comment? It is useful inline documentation

Reply:
Inline comments make impossible to read properly value by envalid library